.eslintignore

```diff
@@ -0,0 +1,3 @@
+elm-stuff
+fixtures
+templates
```

Comment:
Does this have to be its own file or could we include in package.json? I have a slight preference for less file in the top level directory but no stress if this is the only way to exclude files?

Reply:
Feel free to change it if you like.

Reply:
Personally I like it being easy to copy-paste between .gitignore, .eslintignore, .prettierignore, .dockerignore

Reply:
Okay that makes sense. Maybe at some point in the future we can consider merging our config files but this is good for now :)